specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -482,79 +488,79 @@
         "memoryInGB": {
-          "description": "The memory limit in GB of this container.",
+          "description": "The memory limit in GB of this container instance.",
           "type": "number",
           "format": "double"
         },
         "cpu": {
-          "description": "The CPU limit of this container.",
+          "description": "The CPU limit of this container instance.",
           "type": "number",
           "format": "double"
         }
       }
     },
     "AzureFileVolume": {
-      "description": "The Azure file volume.",
+      "description": "The properties of the Azure File volume.",
       "type": "object",
       "required": [
         "shareName",
         "storageAccountName"
       ],
       "properties": {
         "shareName": {
-          "description": "The Azure file share name.",
+          "description": "The name of the Azure File volume.",
           "type": "string"
         },
         "readOnly": {
-          "description": "The flag indicating whether the Azure file volume is read only.",
+          "description": "The flag indicating whether the Azure File volume is read-only.",
           "type": "boolean"
         },
         "storageAccountName": {
-          "description": "The storage account name to access to the Azure file.",
+          "description": "The name of the storage account that contains the Azure File volume.",
           "type": "string"
         },
         "storageAccountKey": {
-          "description": "The storage account key to access to the Azure file.",
+          "description": "The storage account key used to access the Azure File volume.",
           "type": "string"
         }
       }
     },
     "Volume": {
-      "description": "The volume.",
+      "description": "The properties of the volume.",
       "type": "object",
       "required": [
         "name",
         "azureFile"
       ],
       "properties": {
         "name": {
-          "description": "The volume name.",
+          "description": "The name of the volume.",
           "type": "string"
         },
         "azureFile": {
-          "description": "The Azure file volume.",
+          "description": "The Azure File volume.",
           "$ref": "#/definitions/AzureFileVolume"
         }
       }
     },
     "VolumeMount": {
-      "description": "The volume mount.",
+      "description": "The properties of the volume mount.",
       "type": "object",
       "required": [
         "name",
         "mountPath"
       ],
       "properties": {
         "name": {
-          "description": "The volume mount name.",
+          "description": "The name of the volume mount.",
           "type": "string"
         },
         "mountPath": {
-          "description": "The path with in the container at which the volume should be mounted. Must not contain ':'.",
+          "description": "The path within the container where the volume should be mounted. Must not contain ':' (colon).",
```

Comment:
Must not contain colon (:).

(list full character name first, then the character in parens, no quotes)